.github/workflows/subgraph-deploy.yaml

```diff
@@ -1,21 +1,20 @@
+# This workflow is triggered when a pull request is closed on the 'develop' or 'main' branches
+# and there is a change to the 'packages/subgraph/package.json' file.
+
 name: Subgraph Deploy
+
 on:
-  push:
-    branches:
-      - develop
-      - main
-    paths:
-      - 'active_contracts.json'
   pull_request:
     types:
       - closed
     branches:
       - develop
       - main
```

Comment:
Shouldn't there only be one "correct" branch where merges should be monitored?

Reply:
in almost all our flows we do develop and main
once we have decided on main or master we can revisit this 
any way it will not trigger if the correct label is not present